cmd/scheduler/app/server.go

```diff
@@ -56,10 +55,6 @@ import (
 
 // Run the volcano scheduler.
 func Run(opt *options.ServerOption) error {
-	if opt.PrintVersion {
```

Comment:
Hi, welcome! A little question here, when doing secondary development based on volcano and we need to add additional `opt.PrintVersion` when calling `app.Run`, which is not very friendly to developers.

Reply:
So do we add a `PrintVersion` in main.go and also add a `PrintVersion` in Run?